### DISCUSSION TITLE
SKIA WPF schemas

### DISCUSSION THREAD

**Discussion by mwpowellhtx** [mono/SkiaSharp#3004] [Ideas]
Is there such a thing? Short of having to reference namespaces and assemblies themselves.

```
xmlns:skia="clr-namespace:SkiaSharp.Views.WPF;assembly=SkiaSharp.Views.WPF"
```

By contrast some other schema examples, for instance:

```
xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
xmlns:sf="http://schemas.syncfusion.com/wpf"
```

So far the only samples I have been able to identify are [_along these lines_](https://github.com/search?q=repo%3Amono%2FSkiaSharp%20xmlns%3Askia&type=code).

**Comment by mattleibow** [comment]
Good idea, I converted this into an issue: https://github.com/mono/SkiaSharp/issues/3005
